packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -192,6 +192,14 @@ export interface ContainerRuntimeMessage {
     type: ContainerMessageType;
 }
 
+export enum OrderSequentiallyFailureMode {
+    // Close container if callback fails
+    Fail,
+
+    // rollback ops if callback fails
+    Rollback,
+}
```

Comment:
Tony and I discussed an additional value for rollback with fail fast, where the callback would fail as soon as an edit that didn't support rollback happened. (And then possibly the previous ops could be rolled back). I didn't add that since I'm not sure this would actually be useful and I could see DDS implementers getting this wrong easily.

Reply:
similar to above. i think it is too soon to add this. i'd prefer not to add any of this for now, and maybe discuss in an issue what to expose